### PR TITLE
Add SIG Cloud Provider spotlight article.

### PR DESCRIPTION
**Pull request by fsmunoz**
New SIG Spotlight article on SIG Cloud Provider.

Closes: #476

### CODE REVIEW

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+## Introduction
+
+**Arujjwal**: Let's start by getting to know you. Can you give us a small intro about yourself and
+how you got into Kubernetes?
+
+**Michael**: Hi, I’m Michael McCune, most people around the community call me by my handle,
+_elmiko_. I’ve been a software developer for a long time now (Windows 3.1 was popular when I
+started!), and I’ve been involved with open-source software for most of my career. I first got
+involved with Kubernetes as a developer of machine learning and data science applications, the team
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:31]:
```suggestion
involved with Kubernetes as a developer of machine learning and data science applications; the team
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+and when an opportunity arose to join a team working directly on Kubernetes, I jumped at it!
+
+## Functioning and working
+
+**Arujjwal**: Can you give us an insight into what SIG Cloud Provider does and how it functions?
+
+**Michael**: SIG Cloud Provider was formed to help ensure that Kubernetes provides a neutral
+integration point for all infrastructure providers. Our largest task to date has been the extraction
+and migration of in-tree cloud controllers to out-of-tree components. The SIG meets regularly to
+discuss progress, and upcoming tasks, and also to answer questions and bugs that
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:43]:
```suggestion
discuss progress and upcoming tasks and also to answer questions and bugs that
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+arise. Additionally, we act as a coordination point for cloud provider subprojects such as the cloud
+provider framework, specific cloud controller implementations, and the [Konnectivity proxy
+project](https://kubernetes.io/docs/tasks/extend-kubernetes/setup-konnectivity/).
+
+
+**Arujjwal:** After going through the project
+[README](https://github.com/kubernetes/community/blob/master/sig-cloud-provider/README.md), I
+learned that SIG Cloud Provider works with the integration of Kubernetes with Cloud Providers. How
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:51]:
```suggestion
learned that SIG Cloud Provider works with the integration of Kubernetes with cloud providers. How
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+does this whole process go?
+
+**Michael:** One of the most common ways to run Kubernetes is by deploying it to a cloud environment
+(AWS, Azure, GCP, etc). Frequently, the cloud infrastructures have features that enhance the
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:55]:
```suggestion
(AWS, Azure, GCP, etc.). Frequently, the cloud infrastructures have features that enhance the
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+performance of Kubernetes, for example by providing elastic load balancing for Service objects. To
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:56]:
```suggestion
performance of Kubernetes, such as, for example, by providing elastic load balancing for Service objects. To
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+ensure that cloud-specific services can be consistently consumed by Kubernetes, the Kubernetes
+community has created cloud controllers to address these integration points. Cloud providers can
+create their own controllers either by using the framework maintained by the SIG, or by following
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:59]:
```suggestion
create their own controllers either by using the framework maintained by the SIG or by following
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+the API guides defined in the Kubernetes code and documentation. One thing I would like to point out
+is that the SIG cloud provider does not deal with the lifecycle of nodes in a Kubernetes cluster,
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:61]:
```suggestion
is that SIG Cloud Provider does not deal with the lifecycle of nodes in a Kubernetes cluster;
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+for those types of topics SIG cluster lifecycle and the cluster API project are more appropriate
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:62]:
```suggestion
for those types of topics, SIG Cluster Lifecycle and the Cluster API project are more appropriate
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+venues.
+
+## Important subprojects
+
+**Arujjwal:** There are a lot of subprojects within this SIG, can you highlight some of the most
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:67]:
```suggestion
**Arujjwal:** There are a lot of subprojects within this SIG. Can you highlight some of the most
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+important ones and what job they do?
+
+**Michael:** I think the two most important subprojects today are the [cloud provider
+framework](https://github.com/kubernetes/community/blob/master/sig-cloud-provider/README.md#kubernetes-cloud-provider)
+and the [extraction/migration
+project](https://github.com/kubernetes/community/blob/master/sig-cloud-provider/README.md#cloud-provider-extraction-migration). The
+cloud provider framework is a common library to help infrastructure integrators build a cloud
+controller for their infrastructure. This project is most frequently the starting point for new
+people coming to the SIG. The extraction and migration project is the other big subproject and a
+large part of why the framework exists, a little history might help explain further: for a long
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:77]:
```suggestion
large part of why the framework exists. A little history might help explain further: For a long
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+time, Kubernetes has needed to have some integration with the underlying infrastructure, not
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:78]:
```suggestion
time, Kubernetes needed some integration with the underlying infrastructure, not
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+necessarily to add features but to be aware of cloud events like instance termination. The cloud
+provider integrations were built into the Kubernetes code tree, and thus the term "in-tree" was
+created (check out this [article on the topic](https://kaslin.rocks/out-of-tree/) for more
+info). The activity of maintaining provider-specific code in the main Kubernetes source tree was
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:82]:
```suggestion
information). The activity of maintaining provider-specific code in the main Kubernetes source tree was
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+Apache Spark on Kubernetes. That said, I’ve been interested in distributed systems for many years
+and when an opportunity arose to join a team working directly on Kubernetes, I jumped at it!
+
+## Functioning and working
+
+**Arujjwal**: Can you give us an insight into what SIG Cloud Provider does and how it functions?
+
+**Michael**: SIG Cloud Provider was formed to help ensure that Kubernetes provides a neutral
+integration point for all infrastructure providers. Our largest task to date has been the extraction
+and migration of in-tree cloud controllers to out-of-tree components. The SIG meets regularly to
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:42]:
```suggestion
and migration of in-tree cloud controllers to out-of-tree components (more on those terms later). The SIG meets regularly to
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 14]
```diff
@@ -0,0 +1,143 @@
...
+considered undesirable by the community. The community’s decision inspired the creation of the
+extraction and migration project to remove the "in-tree" cloud controllers in favour of
+"out-of-tree" components.
+
+
+**Arujjwal:** What makes it a good place to start? Does it have consistent good beginner work? What
+kind?
+
+**Michael:** I feel that the cloud provider framework is a good place to start as it encodes the
+community’s preferred practices for cloud controller managers, and as such will give a newcomer a
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:92]:
```suggestion
community’s preferred practices for cloud controller managers and, as such, will give a newcomer a
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 15]
```diff
@@ -0,0 +1,143 @@
...
+strong understanding of how and what the managers do. Unfortunately, there is not a consistent
+stream of beginner work on this component, this is due in part to the mature nature of the framework
+and that of the individual providers as well. For folks who are interested in getting more involved,
+having some [Go language](https://go.dev/) knowledge is good, and also having an understanding of
+how at least one cloud API (eg. AWS, Azure, GCP) works is also beneficial. In my personal opinion,
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:97]:
```suggestion
how at least one cloud API (e.g., AWS, Azure, GCP) works is also beneficial. In my personal opinion,
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 16]
```diff
@@ -0,0 +1,143 @@
...
+being a newcomer to SIG cloud provider can be challenging as most of the code around this project
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:98]:
```suggestion
being a newcomer to SIG Cloud Provider can be challenging as most of the code around this project
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 17]
```diff
@@ -0,0 +1,143 @@
...
+deals directly with specific cloud provider interactions. My best advice to people wanting to do
+more work on cloud providers is to grow your familiarity with one or two cloud APIs and then look
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:100]:
```suggestion
more work on cloud providers is to grow your familiarity with one or two cloud APIs, then look
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 18]
```diff
@@ -0,0 +1,143 @@
...
+for open issues on the controller managers for those clouds, and always communicate with the other
+contributors as much as possible.
+
+## Accomplishments
+
+**Arujjwal:** Can you share about an accomplishment(s) of the SIG that you are proud of?
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:106]:
```suggestion
**Arujjwal:** Can you share an accomplishment of the SIG that you are proud of?
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 19]
```diff
@@ -0,0 +1,143 @@
...
+
+**Michael:** Since I joined the SIG, more than a year ago, we have made great progress in advancing
+the extraction and migration subproject. We have moved from an alpha status on the defining
+[KEP](https://github.com/kubernetes/enhancements/blob/master/keps/README.md) to a beta status and
+are inching ever closer to removing the old provider code from the Kubernetes source tree. I've been
+really proud to see the active engagement from our community members and to see the progress we have
+made towards extraction, I have a feeling that within the next few releases, we will see the final
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:113]:
```suggestion
made towards extraction. I have a feeling that, within the next few releases, we will see the final
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 20]
```diff
@@ -0,0 +1,143 @@
...
+removal of the in-tree cloud controllers and the completion of the subproject.
+
+## Advice for new contributors
+
+**Arujjwal:** Is there any suggestion or advice for new contributors on how they can start at SIG
+Cloud Provider?
+
+**Michael:** This is a tricky question in my opinion. SIG cloud provider is focused on the code
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:121]:
```suggestion
**Michael:** This is a tricky question in my opinion. SIG Cloud Provider is focused on the code
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 21]
```diff
@@ -0,0 +1,143 @@
...
+pieces that integrate between Kubernetes and an underlying infrastructure. It is very common, but
+not necessary, for members of the SIG to be representing a cloud provider in an official capacity. I
+recommend that anyone interested in this part of Kubernetes should come to an SIG meeting to see how
+we operate and also to study the cloud provider framework project. We have some interesting ideas
+for future work, such as a common testing framework, that will cut across all cloud providers and
+will be a great opportunity for anyone looking to expand their Kubernetes involvement.
+
+**Arujjwal:** Are there any specific skills you're looking for that we should highlight? To give you
+an example from our own SIG Contribex: if you're an expert in [Hugo](https://gohugo.io/), we can
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:130]:
```suggestion
an example from SIG Contributor Experience: If you're an expert in [Hugo](https://gohugo.io/), we can
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 22]
```diff
@@ -0,0 +1,143 @@
...
+always use some help with k8s.dev!
+
+**Michael:** The SIG is currently working through the final phases of our extraction and migration
+process, but we are looking toward the future and starting to plan what will come next. One of the
+big topics that the SIG has discussed is testing. Currently, we do not have a generic common set of
+tests that can be exercised by each cloud provider to confirm the behaviour of their controller
+manager. If you are an expert in Ginkgo and the Kubetest framework, we could probably use your help
+in designing and implementing the new tests.
+
+This is where the conversation ends. I hope this gave you some insights about SIG Cloud Provider's
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:140]:
```suggestion
---

This is where the conversation ends. I hope this gave you some insights about SIG Cloud Provider's
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 23]
```diff
@@ -0,0 +1,143 @@
...
+aim and working. This is just the tip of the iceberg. To know more and get involved with SIG Cloud
```
Comment (nimbinatus) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:141]:
```suggestion
goals and process. This is just the tip of the iceberg. To know more and get involved with SIG Cloud
```

**Review by nimbinatus — CHANGES_REQUESTED** [review]
Some proofing and suggestions

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 13 above)
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:42]: Why this change?

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 4 above)
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:55]: Why this change?

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 5 above)
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:56]: Why this change?

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 10 above)
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:77]: I'd do this:
```suggestion
large part of why the framework exists. A little history might help explain further: for a long
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 24]
```diff
@@ -0,0 +1,143 @@
...
+and the [extraction/migration
+project](https://github.com/kubernetes/community/blob/master/sig-cloud-provider/README.md#cloud-provider-extraction-migration). The
+cloud provider framework is a common library to help infrastructure integrators build a cloud
+controller for their infrastructure. This project is most frequently the starting point for new
+people coming to the SIG. The extraction and migration project is the other big subproject and a
+large part of why the framework exists, a little history might help explain further: for a long
+time, Kubernetes has needed to have some integration with the underlying infrastructure, not
+necessarily to add features but to be aware of cloud events like instance termination. The cloud
+provider integrations were built into the Kubernetes code tree, and thus the term "in-tree" was
+created (check out this [article on the topic](https://kaslin.rocks/out-of-tree/) for more
```
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:81]: Why this change? It doesn't look like idiomatic English.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 12 above)
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:82]: Why this change?

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 25]
```diff
@@ -0,0 +1,143 @@
...
+info). The activity of maintaining provider-specific code in the main Kubernetes source tree was
+considered undesirable by the community. The community’s decision inspired the creation of the
+extraction and migration project to remove the "in-tree" cloud controllers in favour of
```
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:84]: This is quoting from an interview, so we use the speakers' words.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md [hunk 26]
```diff
@@ -0,0 +1,143 @@
...
+"out-of-tree" components.
+
+
+**Arujjwal:** What makes it a good place to start? Does it have consistent good beginner work? What
```
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:88]: Why this change?

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 17 above)
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:100]: Why this change?

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 18 above)
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:106]: Why this change?

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 21 above)
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:130]: “SIG ContribEx” if that's what was said at the time.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 23 above)
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:141]: Why this change?

**Review by sftim — COMMENTED** [review]
@elmiko and Arrujuval, thank!

---

@nimbinatus thanks for the review. You've suggested a set of changes to the words that the interviewers spoke; we usually wouldn't make that kind of edit.

**Comment by nimbinatus** [comment]
> @nimbinatus thanks for the review. You've suggested a set of changes to the words that the interviewers spoke; we usually wouldn't make that kind of edit.

@sftim my apologies; I may have misunderstood a thread in the contributor comms slack channel when I asked if edits are requested along with a proofread and tech check. I'll drop the ones that change words.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 13 above)
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:42]: I suggested it to make clearer to a new reader that the terms would be addressed later in the discussion. I can add it as an editor's note (square brackets), or perhaps a footnote would be helpful.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 4 above)
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:55]: Proofing fix. `etc.` is an abbreviation for `et cetera`, and it appears in all dictionaries and style guides that I'm aware of (Chicago, APA, MLA) with the period required, even if it's at the end of a sentence (e.g., https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Punctuation/faq0040.html#:~:text=An%20abbreviation%20that%20ends%20with%20a%20period%20must%20not%20be%20left%20hanging%20without%20it%20(in%20parentheses%2C%20e.g.)%2C).

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 5 above)
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:56]: I was trying to make the sentence clearer for a reader. However, referencing my note on the PR, I may have misunderstood how much editing was necessary. This one can be dropped (though the comma after `for example` should stay as the phrase is a parenthetical).

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 10 above)
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:77]: Depends on the style guide we're following and which English usage we would like to follow. American usage usually capitalizes complete sentences; British usage typically does not. Happy to do either one.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 24 above)
[the remark this answers is not in the document]
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:81]: It's a complete sentence inside of the parentheses that's at the end of a sentence. This one can be argued in many style guides, though, so I'm happy to drop it if you'd like.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 12 above)
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:82]: If someone who does not speak English natively were to read this, they can find `information` in most dictionaries but probably won't find `info`.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 26 above)
[the remark this answers is not in the document]
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:88]: Clarity of the antecedent. The `it` here doesn't have a clear connection (is `it` the project? The SIG?), and that issue makes the text hard to follow, especially for an English language learner.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 17 above)
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:100]: This is a list (`grow your familiarity..., look for open issues..., and always communicate`). If you'd rather not change the text, feel free to reject the suggestion.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 18 above)
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:106]: This may have been a misunderstanding of how deep of an edit folks would like on these types of blogs. I was making it cleaner to read.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 21 above)
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:130]: Fair enough. I'd still add an editor's note, like as follows or similar, to make it easier for the reader who isn't familiar with the SIGs:
```suggestion
an example from our own SIG Contribex _[ed. - SIG Contributor Experience]_: If you're an expert in [Hugo](https://gohugo.io/), we can
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 23 above)
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:141]: Parallel structure, mainly. Happy to drop it if you'd rather not use it.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 2 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:43]: now that you mentioned it, what about:

`... discuss progress, upcoming tasks, and also to answer questions...` , keeping the Oxford comma but dropping an "and"?

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 4 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:55]: I'm leaving this open since I completely agree that those style guides support the change, but I've learned and grown used to not using full stops in "etc" inside parenthesis, and putting it after the closing one. I suspect this is a matter of usage and different style guides (specifically the Oxford Style Guide seems to abhor full stops), perhaps with a regional bias.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 10 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:77]: @nimbinatus yes that's an excellent point. I'll go without the capitalisation if that's ok.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 24 above)
[the remark this answers is not in the document]
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:81]: Raising a different topic here... @sftim , is the link to that blog page ok? I seem to remember something about linking to external resources and wouldn't want this to be an issue later.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 25 above)
[the remark this answers is not in the document]
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:84]: @nimbinatus  this one is on me, Michael uses US spelling but it got changed in a previous step (I just checked it). It's the only one that I've found where this happened so I'm going to revert it.

Comment (fsmunoz) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:84]:
```suggestion
extraction and migration project to remove the "in-tree" cloud controllers in favor of
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 15 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:97]: I would do it as "(eg AWS...)" :D I think here it aligns with the previous change though, and Michael used "(e.g. )" before as well.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 17 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:100]: I think that the change makes sense here, IMO. At least, it flows better when I read it.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 18 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:106]: I'll ping the author here.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 21 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:130]: I think we can go with the suggestion (I'll confirm with Arujjwal). I slightly changed this since the original had only "our group" which was not something readers would know about. I'll come back to this one later.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 24 above)
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:81]: I think this link is fine. We don't have a clear blog style guide explanation of why, but I'm happy with it.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 26 above)
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:88]: I recommend: use parens and italics to add editor's notes.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 21 above)
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:130]: Another approach: hyperlink to SIG ContribEx. That's my recommendation.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 23 above)
Comment (sftim) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:143]: @fsmunoz whichever you prefer, then.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 21 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:130]: Tha'ts even better. Will take that one.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 23 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:144]: I'll check with the author.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 2 above)
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:43]: In this case, the SIG meets to discuss two things (progress and tasks) and answer two things (questions and bugs), so there's not a list here in the same way you're thinking. Because there's no list, there's no need for commas. Also, because the second part of the sentence here (`also to answer questions and bugs that arise`) cannot stand alone as a complete thought (an independent clause), it doesn't warrant a comma as if it were a compound sentence.

**Comment by fsmunoz** [comment]
> Like I said, just a misunderstanding :) No harm, no foul. I'll know for the future.

I wouldn't even go that far, since 80%+ of your suggestions are in, so it's already been very useful. Some of the rest are in areas where we likely should have clearer standards (at least, written ones).

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 4 above)
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:55]: I just checked, and yes, the Oxford Style Guide does omit the period after etc. (though, oddly, the Oxford Academic journals use MLA, which does require a period after the abbreviation there). In this case, I'd say it's important to be clear whether the usage intended is British usage or American usage when it comes to editing. Since there's other British usage throughout the doc, I'll drop this.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 26 above)
[the remark this answers is not in the document]
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:88]: Fair enough.

```suggestion
**Arujjwal:** What makes [the cloud provider framework] a good place to start? Does it have consistent good beginner work? What
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 23 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:144]: I've checked with Arrujuval and he would prefer to keep things as-is.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 4 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:55]: @nimbinatus you raise valid concerns and I think this is something that we should discuss more broadly. Especially considering that interviews often involve people who are not native English speakers (and different countries will teach different usages as part of the curriculum), or who use different variants: should we edit it in a way that makes it uniform (and this goes into the details you are mentioning, which are well-sourced and pertinent), or leave them as in the original? 

@sftim I think it would be interesting to have a broader debate on this, and put something into writing (unless there already is and I'm unware of it).

Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:55]: In this specific instance, and to avoid delaying this, I will keep it as-is because that's the way it was originally written - but I agree that it should be ideally changed to something uniform, otherwise we risk having "eg", "eg.", "e.g." and "e.g.," in a simple article.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 23 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:144]: Keeping as-is due to author preference.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 2 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:43]: @nimbinatus yes I agree, thanks for the clarification.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 5 above)
Comment (fsmunoz) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:56]:
```suggestion
performance of Kubernetes, for example, by providing elastic load balancing for Service objects. To
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 10 above)
Comment (nimbinatus) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md]: Fine by me :)

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 5 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:56]: @nimbinatus will add the suggested comma to keep the changes smaller then.

Comment (fsmunoz) [inline content/en/blog/2024/sig-cloud-provider-spotlight.md:56]:
```suggestion
performance of Kubernetes, for example, by providing elastic load balancing for Service objects. To
```

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 18 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:106]: Author prefers to keep original version.

**Comment by fsmunoz** [comment]
@nimbinatus @sftim I incorporated a lot of the suggested reviews, did some small changes as per the discussion, and kept others as-is. I think we have just one left, will pick it up tomorrow.

File: content/en/blog/2024/sig-cloud-provider-spotlight.md (hunk 13 above)
Comment (fsmunoz) [reply content/en/blog/2024/sig-cloud-provider-spotlight.md:42]: Will leave it as-is, I think the clarification makes complete sense but the interview is small enough that adding editor's notes just for this is not needed.

**Comment by sftim** [comment]
Can we bump the publication date (for both articles) back to the 1st of March though?

**Comment by fsmunoz** [comment]
@sftim on it as we speak :)

**Comment by fsmunoz** [comment]
@sftim done!

**Review by nimbinatus — APPROVED** [review]

**Comment by nimbinatus** [comment]
Waiting to be sure @sftim is good

/hold

**Comment by sftim** [comment]
GitHub approve counts as approval @nimbinatus

**Comment by sftim** [comment]
/hold cancel
(may as well)

**Comment by nimbinatus** [comment]
> GitHub approve counts as approval @nimbinatus

Bugger. Sorry about that.
